README.md

```diff
@@ -52,30 +52,71 @@ yet compatible with the golang BOSH v2.
 Required to create certificates. Requires golang.
 ```sh
 go get github.com/square/certstrap
+cd $GOPATH/src/github.com/square/certstrap
+go build
+go install
+```
+
+### stampy
+
+[TODO] What is it needed for?
+
+Requires golang.
+```sh
+go get github.com/SUSE/stampy
+cd $GOPATH/src/github.com/SUSE/stampy
+make
+cp build/linux-amd64/stampy $GOBIN/stampy
 ```
 
 ## Building
 
-1. Run `generate-certs.sh` to generate the SSL certificates required.  The
-    default options are fine.
+1. Get all submodules:
+  ```sh
+    git submodule update --init --recursive
+  ```
 
-2. Create the BOSH release for `cf-mysql`, `uaa`, and `hcf`:
-    ```
-    bosh create release --dir src/cf-mysql-release --force --name cf-mysql
-    bosh create release --dir src/uaa-release --force --name uaa
-    bosh create release --dir src/hcf-release --force --name hcf
-    ```
+1. Load needed environment variables:
 
-3. Build fissile images
-    ```
-    STEMCELL=splatform/fissile-stemcell-opensuse:42.2-0.g58a22c9-28.16
-    fissile build packages --stemcell ${STEMCELL}
-    fissile build images   --stemcell ${STEMCELL}
+   ```sh
+     . .envrc
+   ```
+
+1. Generate the SSL certificates required:
+
+    ```sh
+      make certs
     ```
 
+1. Create the releases:
+
+   ```sh
+     make releases
+   ```
+
+1. Get the opensuse stemcell:
+
+   ```sh
+     docker pull splatform/fissile-stemcell-opensuse:42.2-6.ga651b2d-28.33
```

Comment:
Should probably use `${FISSILE_STEMCELL}` because it'll change some day.

Oh, you're using an explicit version later too, instead of the one from `.envrc`? Okay…